### DISCUSSION TITLE
关于apple相关规则的讨论

### DISCUSSION THREAD

**Discussion by jh0072110** [Openwrt-Passwall/openwrt-passwall2#91] [General]
从psw1代开始就把apple.com放在了直连（direct）里，我就有些困惑了，如果不放在直连里是会影响什么相关服务嘛？（我暂时好像没有遇到）
我是觉得不知道的东西最好保持默认，但保持默认会造成一个问题，就是Apple TV+服务几乎不可用……
Apple TV+服务需要在proxy规则里加入：
itunes.apple.com
mzstatic.com
tv.apple.com
这三条，可是优先级低于direct所以基本不生效？所以我…稍微有些困惑……
或者有大佬们有更好的解决方式希望能不吝赐教……

**Comment by handsomeji** [comment]
可以修改的，没有强制性

**Reply by jh0072110** [reply]
嗯嗯 我只是想知道当初把它放在direct里…是有什么特殊意义么哈哈哈哈

**Reply by chika0801** [reply]
> 嗯嗯 我只是想知道当初把它放在direct里…是有什么特殊意义么哈哈哈哈

https://github.com/Loyalsoldier/v2ray-rules-dat#geositedat-1

geosite:apple-cn：包含 [@felixonmars/dnsmasq-china-list/apple.china.conf](https://github.com/felixonmars/dnsmasq-china-list/blob/master/apple.china.conf) 文件里的域名，供希望 Apple 域名直连（不走代理）的用户使用。

参考上面

**Comment by bamcope** [comment]
感谢分享，按照该方法在Mac Mini做网关模式的Surge建了一个Apple TV+ 的专门分组，用于和普通的Apple服务分组区分开，现在打开Apple TV+是出国的，而iCloud选择了DIRECT，非常丝滑，目前没有出现撞车的问题

[Proxy Group]
🍎 苹果服务 = select, DIRECT, ……
🍎 Apple TV Plus = select, DIRECT, ……

[Rule]
DOMAIN-SUFFIX,itunes.apple.com,"🍎 Apple TV Plus"
DOMAIN-SUFFIX,mzstatic.com,"🍎 Apple TV Plus"
DOMAIN-SUFFIX,tv.apple.com,"🍎 Apple TV Plus"
RULE-SET,https://raw.githubusercontent.com/ACL4SSR/ACL4SSR/master/Clash/Apple.list,"🍎 苹果服务","update-interval=86400"
